tests/unit_tests/models/test_adapter_export.py

```
@@ -100,6 +103,61 @@ class FakeLoRA:
+            self.experts = _ToyExperts()
+
+
+class _ToyLayer(torch.nn.Module):
+    def __init__(self, with_packed_experts: bool = False):
+        super().__init__()
+        self.self_attn = _ToySelfAttention()
+        self.mlp = _ToyMLP(with_packed_experts=with_packed_experts)
+
+
+class _ToyAdapterModel(torch.nn.Module):
+    _keys_to_ignore_on_load_unexpected = [r"^mtp.*"]
```

Ruff (0.15.10)

[warning] Mutable default value for class attribute

(RUF012)